cpp/arcticdb/processing/aggregation.cpp

```diff
@@ -161,4 +161,43 @@ SegmentInMemory MeanAggregatorData::finalize(const ColumnName& output_column_nam
     }
     return res;
 }
+
+void CountAggregatorData::aggregate(const std::optional<ColumnWithStrings>& input_column, const std::vector<size_t>& groups, size_t unique_values) {
+    if(input_column.has_value()) {
+        input_column->column_->type().visit_tag([&] (auto type_desc_tag) {
+            using TypeDescriptorTag =  decltype(type_desc_tag);
+            using RawType = typename TypeDescriptorTag::DataTypeTag::raw_type;
+
+            aggregated_.resize(unique_values);
+
```

Comment:
I was more meaning something like:
```
if constexpr (is_floating_point_type(TypeDescriptorTag::DataTypeTag::data_type)) {
    // What you have below, without the is_floating_point_type checks in the if
} else {
    for (auto group: groups) {
        auto& val = aggregated_[group];
        ++val;
    }
}
```
The constexpr is important to make this a compile time rather than runtime check